### ISSUE TITLE
"Open" or "Web" Annotation

### ISSUE THREAD

**Issue by azaroth42** [w3c/web-annotation#46]
There's a mix of the two in the specification documents and we should be consistent.

Pro "Open":
- There's already brand name recognition 
- There's already implementations and usage of the ontology, which we have updated rather than re-designed.
- The namespace is ns/oa# and it would be good not to change it [c.f. danbri and foaf]

Pro "Web":
- The WG is the Web AWG, not the Open AWG.
- The protocol and client APIs are not derived from the CG work

**Comment by azaroth42** [comment]
From @AFBarstow's PR #43.
My personal (non-chair, bla bla bla) preference is for "Open".

**Comment by tilgovi** [comment]
I sort of prefer Web for the reasons listed in the pro and the fact that I think it's more meaningful. Open is a pretty overloaded term.

**Comment by tilgovi** [comment]
Although, I am very sympathetic to the namespace staying the same argument.

**Comment by shepazu** [comment]
I prefer "Web".

The Open Annotation Data Model has been out for a while, and we aren't yet done with the Web Annotation Data Model spec. 

There may be even more changes between the existing Open Annotation Data Model and the ongoing Web Annotation Data Model, and I think there is value in drawing the distinction between the 2 as different implementation targets. This gives us more latitude to change the data model without disrupting existing implementations.

Plus, I agree with Randall's opinion around the overloading of "Open".

**Comment by iherman** [comment]
There have already been discussions that may lead to (minor, but nevertheless existing) changes in spec. On the other hand, the WG's document has no official trace back to the OADM, and this may lead to confusion. Consequently, changing the name of the document seems to be a cleaner way forward.

I am sensitive to the namespace issue but, in fact, the same issue applies. The namespace may change (maybe some domain or range specifications will change, for example), so having a new namespace set at the beginning of the work is a small investment that may pay off in the future. RDF based implementations can simply change a URI in the namespace declaration, nobody forces them to change the prefix, ie, it does not seem to be a hugely expensive move (though indeed a pain).

Bottom line: I am in favour of the change, even if I realize it is not an obvious move.

**Comment by azaroth42** [comment]
Seems like consensus! :)

**Comment by fhirsch** [comment]
+1 to change

agree with Doug reasoning, especially as we can expect normative statements to change as we proceed.

might want to consider how web annotations will interoperate with older open annotation systems - sounds like much will interoperate (bringing old into new systems, probably less from new to old) Sounds like a Note at some point.

**Comment by azaroth42** [comment]
From Telco on 2015-07-08, decision to use Web Annotation consistently as the name.  Deferring decision regarding changing or keeping the namespace.

**Comment by raydAtLC** [comment]
Based on today’s discussion (on the call)  in my opinion keeping the namespace is the way to go.  Tim’s argument convinced me: While the new spec will not be backward compatible – which is the argument for a new namespace -two namespaces could result in two divergent specs, while a single namespace is more likely to encourage those who have already implemented the old spec to make the necessary adjustments. Rob’s observation that the old spec is only a draft strengthens the argument to keep the namespace.

**Comment by tilgovi** [comment]
Darn. That's not the worst thing, but it is a little unfortunate if the namespace is "oa" but it's totally not obvious why without a history lesson. :-/

Ah, well. It's no worse than "referer".

**Comment by shepazu** [comment]
Hadn't we previously discussed moving all our namespaces to schema.org? Having a single community namespace for common vocabularies, while suboptimal from a decentralization point of view, makes it easier on developers. We tried distributed namespaces for over a decade, and it led to confusion and disuse.

**Comment by raydAtLC** [comment]
Doug, can you provide a pointer to that discussion.  I cannot imagine why anyone would think it's a good idea to move "all our namespaces to schema.org" but maybe the discussion would shed some light on the idea.

**Comment by azaroth42** [comment]
It has certainly been floated, normally by schema.org proponents, that namespaces are confusing, despite them being used practically everywhere, in all aspects of computer and information science. Having libraries/modules/packages/gems/whatever to distinguish the components included from other components with the same name in other l/m/p/g/w seems something we have all accepted as fundamental long, long ago.

That said, if schema.org is willing to include all of our properties that we need, I'm totally fine with that. I suspect that's not going to be the case, but @danbri or others more involved with that work would be better placed to comment.

**Comment by azaroth42** [comment]
Suggest continuing the NS discussion in the new #53 for that purpose, so we can close this issue once the editorial work is one. Gracias!
